### DISCUSSION TITLE
Email validation

### DISCUSSION THREAD

**Discussion by GoldieBowyien** [GoldieBowyien/Movie_streaming#9] [Q&A] [answered]
What library in JavaScript do use to validate email addresses ?

**Comment by NeoDev99** [comment, marked as answer]
Isemail is an email address validation library for Node. js. It helps you to validate the IDs according to RFCs 5321, 5322, and others.

**Comment by pheneus** [comment]
var validator = require("email-validator");
 
validator.validate("test@email.com"); // true
